src/Services/PayService/PayService.cs

```diff
@@ -108,7 +104,7 @@ public async Task<string> ProcessPaymentResponse(string form, string responseCod
 
         public async Task<PaymentInformation> GetFormPaymentInformation(MappingEntity formData, string form, Page page)
         {
-            var paymentConfig = await _cache.GetFromCacheOrDirectlyFromSchemaAsync<List<PaymentInformation>>($"paymentconfiguration.{_hostingEnvironment.EnvironmentName}", _distributedCacheExpirationConfiguration.PaymentConfiguration, ESchemaType.PaymentConfiguration);
+            var paymentConfig = await _paymentConfigProvider.Get<List<PaymentInformation>>();
             var formPaymentConfig = paymentConfig.FirstOrDefault(_ => _.FormName == form);
```

Comment:
can you just  check if _cache is used anywhere else in the project if not can be removed,i believe it is not anymore, might be wrong.

Reply:
Looks like it was used in PageHelper at one point, but now is unused. I'll remove it and the tests too